Extensions/IISWebAppDeploy/Src/vss-extension.json

```diff
@@ -33,11 +33,10 @@
       "path": "Tasks/IISWebAppDeploy"
     },
     {
-      "path": "images",
-      "addressable": true
```

Comment:
it is not required, I didn't use for other paths as well still it is working. So I removed it.